### PR TITLE
feat: Adding calls to custom style hooks derived from context in react-spinner components

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Changes
- Updated components to call custom style hook

## Issues
- Updates #27058

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                             |   Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                      Change |
| :-------------------------------------------------------------------------------------------- | -------------------------: | --------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr> | `20.62 kB`<br />`6.682 kB` | `23.377 kB`<br />`7.189 kB` | `2.757 kB` <br />`507 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `67.061 kB`<br />`18.105 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `205.644 kB`<br />`57.056 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `37.891 kB`<br />`12.039 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.324 kB`<br />`2.129 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/efb2274e37c0237f3a10a4630b2696a9880f6398'>efb2274e37c0237f3a10a4630b2696a9880f6398</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Avatar-mount_base.html">1270</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Avatar-mount.html">1270</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Button-mount_base.html">925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Button-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Field-mount_base.html">1970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Field-mount.html">1905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProvider-mount_base.html">1553</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProvider-mount.html">1554</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProviderWithTheme-mount_base.html">588</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProviderWithTheme-mount.html">578</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">553</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">579</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/InfoButton-mount_base.html">521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/InfoButton-mount.html">520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/MakeStyles-mount_base.html">1981</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/MakeStyles-mount.html">1942</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Persona-mount_base.html">2833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/Persona-mount.html">2803</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/SpinButton-mount_base.html">2290</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27075/perf-test-react-components/SpinButton-mount.html">2287</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: change/@fluentui-react-spinner-fd1e12fa-8110-48a3-8385-8daaa4f3c09c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: custom styles",
```
Comment (khmakoto) [inline change/@fluentui-react-spinner-fd1e12fa-8110-48a3-8385-8daaa4f3c09c.json:3]:
```suggestion
  "comment": "feat: Adding calls to custom style hooks derived from context.",
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**